.github/workflows/amp_deploy.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy AMP integrations
+
+on: [push]
```

Comment:
Since we want this to be an example that we can share in our documentation, can you modify it so that:
- it deploys to the test project on every commit in a pull request
- it deploys to the prod project on merge on main.